### PR TITLE
adding component

### PR DESCRIPTION
**Pull request by bipin7yadav**

### CODE REVIEW

File: avatar.css [hunk 1]
```diff
@@ -0,0 +1,124 @@
+body{
+    position: relative;
+}
+.container{
+    background-color:skyblue ;
```
Comment (srishti-maurya) [inline avatar.css]: Suggestion: you can use these values as color variables

File: avatar.css [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    padding: 3px;
+    display: flex;
+    gap: 5px;
```
Comment (srishti-maurya) [inline avatar.css:9]: You may give it in rems maybe

File: avatar.css [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+}
+
+.text-square2{
+    background-color: white;
+    border-radius: 5px;
+    padding: 3px;
+    display: inline;
+}
+
+.text-square3{
```
Comment (srishti-maurya) [inline avatar.css]: All these have same property so may be you can name them `text` and use accordingly instead of repeating it

File: avatar.css [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    padding: 3px;
+    display: inline;
+}
+
+.btn{
+    display: flex;
+    justify-content: space-between;
+}
+
+button{
```
Comment (srishti-maurya) [inline avatar.css]: You may avoid styling directing the tags and instead use a class for it

**Review by srishti-maurya — COMMENTED** [review]
You may add a live preview link or gif of the UI for convenience

File: avatar.css (hunk 2 above)
Comment (bipin7yadav) [reply avatar.css:9]: but rem would be to large as 1rem equals to 16px

**Review by manojkumar8400 — APPROVED** [review]
looks good to me to go to merge it.

**Pull request merged**